nunit.sln

```diff
@@ -1,6 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2012
+# Visual Studio 14
```

Comment:
It always seems strange to me that simply opening a solution in a newer version of VS causes changes to be made!